python/default_pyspark_version.txt

```diff
@@ -0,0 +1 @@
+2.4.6
```

Comment:
It seems that putting dependencies and versions in a **requirments.txt** under each submodule dir is more 'python style'? @hkvision 

Others lgtm.

Reply:
Maybe can we put it here?
https://github.com/intel-analytics/BigDL/tree/main/python/requirements/orca
Add a requirement_pyspark.txt?

Reply:
> Maybe can we put it here? https://github.com/intel-analytics/BigDL/tree/main/python/requirements/orca Add a requirement_pyspark.txt?

Sure, but `dllib` you mean? pyspark is binded with dllib.

Reply:
But since dllib doesn't have a requirements folder, if we don't create it, just put it under orca?